styles/stylesheet.css

```diff
@@ -0,0 +1,296 @@
+* {
+  box-sizing: border-box;
+}
+html, body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+body {
+  min-height: 100%;
+  display: grid;
+  grid-template-rows: 1fr auto;
+  font-family: 'Arial', sans-serif;
+}
+body a {
+  text-decoration: none; /* removes hyperlink underlines */
+}
+body a:hover {
+  color: black;
+  text-decoration: underline;
+}
+/********** MEDIA QUERIES **********/
+/* for phones - dog grid layout needs to change from 3 columns to 1 (home) and hide blog aside */
+@media not print and (min-width: 576px) and (max-width: 767px) {
```

Comment:
I suggest changing all of your media queries from `@media not print` to `@media only screen`. You should then notice your blog preview going away on mobile. I would also suggest removing the minimum width here. There may be mobile devices smaller than 576px. For example, if I stretch my browser window to the smallest, it'll go below your min width here.